python/Ganga/Runtime/bootstrap.py

```diff
@@ -1069,12 +1070,12 @@ def convert_merger_to_postprocessor(j):
 
         # JobTree
         from Ganga.Core.GangaRepository import getRegistry
-        jobtree = GPIProxyObjectFactory(getRegistry("jobs").getJobTree())
+        jobtree = getRegistry("jobs").getJobTree()
```

Comment:
Good point - hadn't noticed that so actually the original `GPIProxyObjectFactory` was unnecessary. Ignore me!